### DISCUSSION TITLE
Shortcut to clear the screen

### DISCUSSION THREAD

**Discussion by dingman** [spatie/ray#297] [Ideas]
I'm finding that the screen scrolls quickly and then fills up, and it's hard to go back and look at something. The ability to clear the screen with a keyboard shortcut would be really nice.

**Comment by freekmurze** [comment]
The shortcut you're looking for is `Cmd+K` (use `ctrl` if you're not on MacOS).

In the preferences you can also define a shortcut to clear the screen from anywhere (Ray doesn't have to be the active app)
